--- a/src/curl/Makefile.am
+++ b/src/curl/Makefile.am
@@ -5,10 +5,11 @@ AM_CPPFLAGS = -I$(top_srcdir)
 
 if WITH_CURL
 
-noinst_LTLIBRARIES = libslurm_curl.la
+lib_LTLIBRARIES = libslurm_curl.la
 libslurm_curl_la_SOURCES = slurm_curl.c slurm_curl.h
 libslurm_curl_la_LIBADD = $(LIB_CURL)
+libslurm_curl_la_LDFLAGS = -version-info @SLURM_API_CURRENT@:@SLURM_API_REVISION@:@SLURM_API_AGE@
 
 force:
 $(libslurm_curl_la_LIBADD) : force
 	@cd `dirname $@` && $(MAKE) `basename $@`
